ci/docker/linux-apt-docs.dockerfile

```diff
@@ -18,23 +18,40 @@
 ARG base
 FROM ${base}
 
+# NOTE: as of 2019-12, R 3.5 and 3.6 are available in the repos with -cran35 suffix
```

Comment:
We don't need these comments here, no one is going to use old R to build the docs.